packages/components/src/search-control/index.tsx

```diff
@@ -14,8 +15,9 @@ import { forwardRef, useRef } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { Button } from '../';
+import { Button } from '../button';
```

Comment:
Without this change, there's an error, probably from `Button` not being in TypeScript:

<details>
<summary>error</summary>
```
File 'plugins/gutenberg/packages/components/src/index.js' is not listed within the file list of project 'plugins/gutenberg/packages/components/tsconfig.json'. Projects must list all files or use an 'include' pattern.
  The file is in the program because:
    Imported via '../' from file '/plugins/gutenberg/packages/components/src/search-control/index.tsx'
```
</details>